projects/packages/forms/src/blocks/contact-form/attributes.js

```diff
@@ -41,13 +40,11 @@ export default {
 		type: 'object',
 		default: {
 			organizationId: '',
-			sendToSalesforce: false,
 		},
 	},
 	mailpoet: {
 		type: 'object',
 		default: {
-			enabledForForm: false,
```

Comment:
We remove the default values here so we can set them based on what is defined in PHP, and filtered by the developer. This also allows us to tell if a form block is newly added (these values will start undefined) or already existing (these values will be set and persisted based on our new defaults).